src/components/ToyList.js

```diff
@@ -42,17 +51,17 @@ const ToyList = ({ toys }) => {
       </div>
       <div className="filterByReviews">
         <Button className="m-2" value="toy" onClick={HandleFilterWithReviews}>
-          Toys with Reviews{" "}
+          Toys with Reviews{' '}
         </Button>
         <Button className="m-2" value="toy" onClick={ResetFilter}>
-          Reset{" "}
+          Reset{' '}
```

Comment:
I'm not sure what this looks like on the button, but was the `{' '}` intentional here?

Reply:
You're right! I got rid of that code and it was not needed.